modules/imaging_uploader/test/TestPlan.md

```diff
@@ -109,3 +109,28 @@
 	archive, answer 'Yes'. Check that this reupload fails with error code 4 (and not 2). 
 	Related to Redmine#14093 and PR#3555.
 	[Manual Testing]
+20. First, set the config setting 'ImagingUploader auto launch' to 'No'. Upload an MRI archive that can successfully be
+    uploaded. Check that the value of the Progress column for the uploaded archive is 'Not started'.
+    [Manual Testing]
+21. First, set the config setting 'ImagingUploader auto launch' to 'Yes'. Upload an MRI archive that can successfully be
+    uploaded and that will be processed successfully by the pipeline. Once the upload is finished, check that the value of 
+    the Progress column for the uploaded archive is 'In Progress...'
+    [Manual Testing]
+22. First, set the config setting 'ImagingUploader auto launch' to 'Yes'. Upload an MRI archive that can successfully be
```

Comment:
This step can be merge with previous (most of the text is duplicate).

Reply:
OK